keyboards/cornia/v1/keyboard.json

```
@@ -0,0 +1,187 @@
+{
+    "keyboard_name": "Cornia v1",
+    "usb": {
+        "pid": "0x1000",
+        "device_version": "1.0.0"
+    },
+    "processor": "RP2040",
+    "bootloader": "rp2040",
+    "features": {
+        "extrakey": true,
+        "nkro": true,
+        "rgblight": true
```

You default here to `rgblight`, but do not configure any animations for it?

```suggestion
        "rgb_matrix": true
```